dev/archery/archery/crossbow/cli.py

```diff
@@ -233,6 +233,27 @@ def latest_prefix(obj, prefix, fetch):
     click.echo(latest.branch)
 
 
+@crossbow.command()
+@click.argument('job-name', required=True)
+@click.pass_obj
+def save_report_data(obj, job_name):
+    """
+    Just print there the state of the job
+    """
+    output = obj['output']
+    
+    queue = obj['queue']
+    print(dir(queue))
```

Comment:
I assume this is a leftover.

Reply:
What's the output of this one? Were you able to confirm that `queue` has the `create_commit` method? In the end  the goal of printing `queue` was just to confirm it's an instance of `Queue` and thus has all `Repo` methods. Once you confirmed that, you can remove this print.

Reply:
now i'm gonna check this, because i made some 'breaking' changes